app/controllers/content_items_controller.rb

```diff
@@ -91,23 +91,42 @@ def load_taxonomy_navigation
                                .map { |document_collection| document_collection.values_at('base_path', 'title') }
 
       # Fetch link attributes of parent step by steps required to render the top navigation banner
-      @banner_items = @content_item
+      @step_by_step_links = @content_item
```

Comment:
We don't need `@step_by_step_links`, or `@taxon_links` to be instance variables, only `@banner_items`.